### PR TITLE
chore(v0.14): add migration utility for v0.14

### PR DESCRIPTION
**Pull request by wass3r**

### CODE REVIEW

File: migrations/v0.14/go.mod [hunk 1]
```diff
@@ -0,0 +1,47 @@
+module github.com/go-vela/community/migrations/v0.14
+
+go 1.17
+
+require (
+	github.com/go-vela/server v0.12.1
+	github.com/go-vela/types v0.12.0
```
Comment (cognifloyd) [inline migrations/v0.14/go.mod:7]: Why v0.12?

**Review by cognifloyd — DISMISSED** [review]
I'm generally ok with this. I have one question, but otherwise this is great.

File: migrations/v0.14/go.mod (hunk 1 above)
Comment (wass3r) [reply migrations/v0.14/go.mod:7]: Copy pasta :shame: - when I tested it didn't affect anything but I'd rather update that. Thanks for catching it.

**Review by cognifloyd — DISMISSED** [review]

**Review by jbrockopp — DISMISSED** [review]
LGTM

**Review by jbrockopp — APPROVED** [review]
LGTM

**Review by KellyMerrick — APPROVED** [review]
lgtm

**Pull request merged**